packages/react-components/src/components/TypingIndicator.tsx

```diff
@@ -16,6 +16,20 @@ export interface TypingIndicatorStylesProps extends BaseCustomStylesProps {
   typingString?: IStyle;
 }
 
+/**
+ * Strings of TypingIndicator that can be overridden
+ */
+export interface TypingIndicatorStrings {
+  /** String to use when one user is typing */
+  singular: string;
+  /** String to use when multiple users are typing */
+  plural: string;
+  /** String to use when multiple users are typing with one other user abbreviated */
+  shortenedPlural: string;
+  /** String to use when multiple users are typing with multiple other users abbreviated */
+  shortenedPlural2: string;
```

Comment:
Can we name this better? How about `singleUser`, `multipleUsers`, `multipleUsersAbbreviateOne`, `multipleUsersAbbreviateMany`?